index.bs

```diff
@@ -937,12 +937,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  If <code>|credentialCreationData|.[=attestationConveyancePreferenceOption=]</code>'s value is
                     <dl class="switch">
                         :   "none"
-                        ::  Replace potentially uniquely identifying information (such as [=AAGUID=] and
-                            [=attestation certificates=]) in the [=attested credential data=] and [=attestation statement=],
-                            respectively, with blinded versions of the same data.
-
-                        Issue(w3c/webauthn#694): need to define "blinding".  See also
-                            [#462](https://github.com/w3c/webauthn/issues/462).
+                        ::  Replace potentially uniquely identifying information with non-identifying versions of the
+                            same. Specifically, implementations SHOULD, at least:
+                               1. Replace the [=AAGUID=] in the [=attested credential data=] with 16 zero bytes.
+                               1. Replace the [=attestation statement=] with a  <a href="#none-attestation">&ldquo;none&rdquo; attestation</a>. Alternatively, replace it with a <a href="#packed-attestation">packed</a> or <a href="#fido-u2f-attestation">U2F</a> attestation containing a single, fresh, self-signed certificate. (If the original attestation used X.509 certificates then any FIDO transports extension [[!FIDO-TRANSPORTS-EXT]] from the original MAY be duplicated into the self-signed replacement.)
```

Comment:
With U2F, we copy the transports extension into the randomly generated attestation certificate because that's still useful information. However, I think I was being too influenced by U2F here: webauthn already conveys the transport in a clean way. So I'm actually leaning towards saying that a &ldquo;none&rdquo; attestation type should always be used rather than a basic attestation.

Self-attestation does not seem immediately applicable because &ldquo;it uses the credential private key to create the attestation signature&rdquo; and thus needs the cooperation of the token. If available, that does make up for a questionable part of the crypto design here, namely that there's no proof of possession otherwise. But it only works if the token sets the AAGUID to zero.

Let me try to rewrite this with that in mind and see how you like it afterwards.